src/certificate.js

```diff
@@ -64,6 +64,17 @@ function deserializeCertFields (buf: Buffer, offset: number) {
     return { cert, offset }
   }
 
+  function getTotalAmount(outputs) {
```

Comment:
Perhaps a more specific function name?  The logic is specific to backward transfers, so maybe `getTotalBtAmount`?